### ISSUE TITLE
Feature: finish_launch updates launch attributes(tags)

### ISSUE THREAD

**Issue by rjswitzer3** [reportportal/client-Python#130]
`finish_launch()` does not provide the ability to update/override launch attributes(tags) upon completion. This feature is supported by the underlying API, but has not been included in the client.

Based on the implementation of `finish_test()`, this seems to be a relatively straightforward implementation.

```
def finish_launch(self, end_time, status=None, attributes=None, **kwargs):
    """Finish a launch with the given parameters.
    Status can be one of the followings:
    (PASSED, FAILED, STOPPED, SKIPPED, RESETED, CANCELLED)
    """
    # process log batches firstly:
    if self._batch_logs:
        self.log_batch([], force=True)
    if attributes and isinstance(attributes, dict):
        attributes = _dict_to_payload(attributes)
    data = {
        "endTime": end_time,
        "status": status,
        "attributes": attributes
    }
    url = uri_join(self.base_url_v1, "launch", self.launch_id, "finish")
    r = self.session.put(url=url, json=data, verify=self.verify_ssl)
    logger.debug("finish_launch - ID: %s", self.launch_id)
    return _get_msg(r)
```

This would be great to have so one could add additional tags based on the results of a launch. Of which would also provide more filtering capabilities in the launch view.

**Comment by iivanou** [comment]
Why don't you create a PR?

**Comment by iivanou** [comment]
Fixed in #131
